src/lib/app-switcher/_app-switcher-theme.scss

```diff
@@ -1,8 +1,13 @@
 @mixin vantage-app-switcher-theme($theme) {
   $foreground: map-get($theme, foreground);
   $text: map-get($foreground, text);
+  $secondary-text: map-get($foreground, secondary-text);
 
   .app-switcher-header-title {
     color: $text;
   }
+  .other-products-header {
+    color: $secondary-text;
+    @include mat-typography-level-to-styles($fontConfig, caption);
```

Comment:
This won't scale because we can't rely on $fontConfig. 3 possible options: 
1. Create a separate mixin for typography
2. We move this into the app-switcher.scss file with a comment mentioning why its there
3. Pass the mat-caption class seems to the component which seems like the simplest way for now.

Reply:
Why can't we depend on angular material's own typography functions? .mat-caption is just an abstraction of this function in the first place.

Reply:
We can, the issue is fontConfig, its not generic. The first two options I suggest are supposed to use material functions/mixins.